internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -214,42 +214,60 @@ func mapPage(vu moduleVU, p *common.Page) mapping { //nolint:gocognit,cyclop
 				return p.InputValue(selector, popts) //nolint:wrapcheck
 			}), nil
 		},
-		"isChecked": func(selector string, opts sobek.Value) *sobek.Promise {
+		"isChecked": func(selector string, opts sobek.Value) (*sobek.Promise, error) {
+			popts := common.NewFrameIsCheckedOptions(p.Timeout())
```

Comment:
Can this be p.MainFrame().Timeout() as it takes slightly different path and is the actual thing that is used underneed.

Reply:
the mappings are called when they are called. So if you get the timeout before that and it changes that will be a problem

Reply:
Done with the latest forced commits